src/pages/workflows/WorkflowList.tsx

```diff
@@ -166,6 +170,30 @@ export const WorkflowList = (props: WorkflowListProps) => {
     loadWorkflows();
   });
 
+  const uploadWorkflow = async () => {
+    try {
+      const workflowPayload = {
+        namespace: workflowNamespace,
+        name: workflowName,
+        synopsis: workflowSynopsis,
+        snapshotComment,
+        documentation: workflowDocumentation,
+        payload: workflowWdl,
+        entityType: 'Workflow', // Currently not supporting task creation
+      };
+
+      const { namespace, name, snapshotId } = await Ajax(signal).Methods.methods(workflowPayload);
+      Nav.goToPath('workflow-dashboard', {
+        namespace,
+        name,
+        snapshotId,
+      });
+    } catch (e) {
+      setCreateWorkflowModalOpen(false);
+      reportError('Error uploading method', e);
```

Comment:
Out of curiosity, does this show a toast notification?

Reply:
Yes!